src/App.tsx

```diff
@@ -1,9 +1,50 @@
+import React from 'react';
+import {
+  NavLink, Routes, Route, Navigate, useParams,
+} from 'react-router-dom';
 import './App.scss';
+import { HomePage } from './components';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/PageNotFound/PageNotFound';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const Redirect = () => {
+  const { type } = useParams();
 
-export default App;
+  return (
+    <>
+      <h2>{type}</h2>
+      {type?.includes('home')
+      && <Navigate to="/" replace />}
+    </>
+  );
+};
+
+export const App: React.FC = () => {
+  return (
+    <div className="App">
+      <nav>
+        <NavLink
+          to="/"
+        >
+          Home
+        </NavLink>
+        {' '}
+        <NavLink
+          to="/people"
+        >
+          People page
+        </NavLink>
+      </nav>
```

Comment:
Maybe this should be a component Header( point 7)(Add a Header visible everywhere with navigation links to both pages)